grassroots-backend/openAPI.json

```diff
@@ -32,37 +32,30 @@
         ]
       }
     },
-    "/auth/login": {
-      "get": {
-        "operationId": "AuthController_login",
+    "/contacts": {
```

Comment:
There's a pretty big diff here. I would have expected openAPI's ordering to be stable. I wonder if we should add a sort to this object.

Reply:
I'll address this in my formatting PR.